gha-webhook-listener.py

```diff
@@ -137,6 +138,14 @@ def on_receive_poke():
         abort(400, "Unrecognised event")
         return
 
+    workflow_status = incoming_json["workflow_run"]["head_branch"]
+    if workflow_status == arg_branch_name:
+        logger.info(f"Workflow was for branch {arg_branch_name}")
+    else:
+        logger.info("Rejecting '%s' event", event)
+        abort(400, "Wrong branch")
```

Comment:
we should just ignore it, and return a 200 rather than a 400?

Reply:
Would this still use `abort`?

Reply:
I don't think so. You just return the body you want to return in the response, same as line 212:

```
    return jsonify({})
```